### PR TITLE
ENH: Add a utility for finding sweep number keys

### PR DESCRIPTION
**Pull request by mgrover1**
Add a utility to find the `sweep_n` keys in a given datatree. This will help with downstream applications, especially with Py-ART, and since this information will be stored in the core data model.

- [x] Tests added
- [x] Changes are documented in `history.md`

### CODE REVIEW

File: xradar/util.py [hunk 1]
```diff
@@ -487,3 +488,30 @@ def rolling_dim(data, window):
...
+    Returns
+    -------
+    keys : list
+        List of associated keys with sweep_n
+    """
+    sweep_group_keys = []
+    for key in list(dt.children):
+        parts = key.split("_")
+        try:
+            # Try to set the second part of the tree key to an int
```
Comment (kmuehlbauer) [inline xradar/util.py:510]: Can we check if the first part equals `sweep`, too?

**Review by kmuehlbauer — APPROVED** [review]
@mgrover1 That looks good to me. Just one comment which could be addressed.

Otherwise, go for it. :+1:

**Comment by mgrover1** [comment]
I'll add that in!! 👍

**Comment by mgrover1** [comment]
@kmuehlbauer good to merge?

**Review by kmuehlbauer — APPROVED** [review]

**Pull request merged**